bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -1222,7 +1222,7 @@ spec:
                   value: /opt/manifests
                 - name: ODH_PLATFORM_TYPE
                   value: OpenDataHub
-                image: REPLACE_IMAGE:latest
+                image: ttl.sh/882e644b-532e-4feb-95d4-2bd23fb2e6df:2h
```

Comment:
I guess this is probably WIP for testing?

Reply:
yep mistake

Reply:
done